src/tasks/dump_db/readme_for_tarball.md

```diff
@@ -0,0 +1,30 @@
+# crates.io Database Dump
```

Comment:
I love this change and the change moving the CSVs to a `data` directory! Really cleans up the `tar` file contents.